include/swift/ABI/ObjectFile.h

```diff
@@ -50,6 +60,11 @@ class SwiftObjectFileFormatMachO : public SwiftObjectFileFormat {
     }
     llvm_unreachable("Section type not found.");
   }
+
+  bool sectionContainsReflectionData(llvm::StringRef sectionName) override {
+    return super::sectionContainsReflectionData(sectionName) ||
+           sectionName == "__DATA_CONST,__const";
```

Comment:
I might be wrong, but can't "__DATA,__const" have reflection information as well? I don't know how the compiler chooses to emit in either of those sections. Other than that, looks good to me!

Reply:
I don't know how long the the behavior has existed, but ld64 turns `__DATA,__const`, as output by the compiler, to `__DATA_CONST,__const`. I believe this is done because permissions and the `SG_READ_ONLY` flag apply at the segment level, not the section. If this behavior is new, then we may need to handle `__DATA,__const` as well.

Reply:
You can test this this by compiling an example for various earlier deployment targets on x86 macOS and arm64(e) iOS.

Reply:
Oh, I forgot: we need to include "__DATA_CONST,__got" as well.

Reply:
I have added `__DATA,__const` here too.